parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestEncryptionOptions.java

```diff
@@ -24,26 +24,10 @@
 
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
+import java.util.*;
```

Comment:
Please avoid import *

Reply:
Sorry. This was done by IntelliJ as part of its (otherwise useful) import optimization feature. I'll try to change the settings.

Reply:
Done.